WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -93,6 +93,14 @@ def index
       end
     end
 
+    begin
+      from_date = params[:from_date] && Date.parse(params[:from_date])
+      @competitions = @competitions.where("start_date >= ?", from_date) if from_date
```

Comment:
Would this be worse in some way I'm missing?

`@competitions = @competitions.where("start_date >= ?", Date.parse(params[:from_date])) if params[:from_date]`